network/reference/ports.go

```diff
@@ -14,8 +14,8 @@ var (
 		25:  "SMTP",
 		43:  "WHOIS",
 		53:  "DNS",
-		67:  "DHCP-SERVER",
-		68:  "DHCP-CLIENT",
+		67:  "DHCP_SERVER",
+		68:  "DHCP_CLIENT",
```

Comment:
Isn't that a breaking change? Rules that use the old port-names will be broken now. Should we do a migration for that?

Reply:
For name to number conversion both versions are supported.